.github/.coderabbit.yaml

```
@@ -0,0 +1,129 @@
+  # --- 자동 리뷰 --------------------------------------------------------------
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    ignore_title_keywords: []
+    labels: []
+    drafts: false
+    base_branches: [ 'main' ]
+
+  # --- 활성화 도구 ------------------------------------------------------------
+  tools:
+    shellcheck:   { enabled: true  }
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)